pubspec.yaml

```diff
@@ -1,40 +1,32 @@
 name: peanut
-version: 3.3.0
```

Comment:
Revert gratuitous churn here, please!

Reply:
Did you run some tool?

Reply:
I ran https://github.com/f3ath/pubspec-version - I'll clean this up